LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ApplicationInsightsMetricExporter.cs

```diff
@@ -35,7 +37,7 @@ public ApplicationInsightsMetricExporter(TelemetryClient telemetryClient, Regist
             this.metricRegistry = registryLookup.ToDictionary(m => m.Key, m =>
             {
                 var id = new MetricIdentifier(MetricRegistry.Namespace, m.Value.Name, m.Value.Tags);
-                return (telemetryClient.GetMetric(id), id);
+                return (telemetryClient.GetMetric(id), id.GetDimensionNames().ToArray());
```

Comment:
unrelated change, but fixed as part of this PR since I touched the code in any case